examples/k8s_monolith/external_nsc/README.md

```diff
@@ -0,0 +1,39 @@
+# NSC as external docker container
```

Comment:
I am curious why this is in the k8s_monolith example?  We should be able to use  simple docker nsc as a client for any interdomain Network Service use case, not just the breaking up the monolith case.